elephant/statistics.py

```diff
@@ -13,8 +13,11 @@
 import scipy.stats
 import scipy.signal
 import neo
-import warnings
+from neo.core import SpikeTrain
 import elephant.conversion as conv
+import kernels
```

Comment:
Please write `import elephant.kernels as kernels`. This then should work also with the python3 builds of travis.